dts/arm/unisoc/uwp5661.dtsi

```diff
@@ -0,0 +1,60 @@
+#include <arm/armv7-m.dtsi>
+
+#include <unisoc/mem.h>
+
+#define INT_UART0	35          // UART0 Rx and Tx
+#define INT_UART1	36          // UART1 Rx and Tx
+#define INT_UART2	7           // UART2 Rx and Tx (Connect EIC0(7))
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m4";
+			reg = <0>;
+		};
+	};
+
+	sram0: memory@100000 {
+		device_type = "memory";
+		compatible = "mmio-sram";
+		reg = <DT_SRAM_START DT_SRAM_SIZE>;
+		   };
+	flash0: flash@200000 {
+		compatible = "serial-flash";
+		reg = <DT_FLASH_START DT_FLASH_SIZE>;
+	};
+
+	soc {
+		uart0: uart@40038000 {
+			compatible = "unisoc,uart";
```

Comment:
Is "unisoc,uart" too generic?  Do you think other future SoCs will use the exact same UART hardware?

Reply:
have rename to unisoc,uwp-uart